.eslintrc.json

```diff
@@ -1,24 +1,11 @@
-{
-  "root": true,
-  "extends": [
-    "airbnb-base",
-    "airbnb-base",
-    "prettier",
-    "airbnb/hooks",
-    "plugin:mocha/recommended",
-    "plugin:jest/recommended",
-    "prettier",
-    "plugin:prettier/recommended"
-  ],
-  "plugins": ["node", "prettier", "eslint-plugin-node", "jest"],
```

Comment:
explain why you removed plugins.

Reply:
check on the other rules they gave us in pivot tracker there were no plugins 
and another thing those rules are not supported by hound